<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyGUI</class>
 <widget class="QMainWindow" name="PyGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Input data</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="temperature_ouput">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current temperature</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Parameterization</string>
    </property>
    <widget class="QComboBox" name="comboBox_temp_unit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Kelvin [ K ]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Celsius [ °C ]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fahrenheit [ °F ]</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="sampling_time_input">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sampling time in ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Unit of temperature</string>
     </property>
    </widget>
    <zorder>sampling_time_input</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>comboBox_temp_unit</zorder>
   </widget>
   <widget class="QPushButton" name="button_update">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>button_update</zorder>
   <zorder>button_start</zorder>
   <zorder>button_stop</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication_by_Darren_Jonas_and_Finn">
    <property name="title">
     <string>Temperature Application</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
   </widget>
   <addaction name="menuApplication_by_Darren_Jonas_and_Finn"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
